docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,613 @@
+# Deploying Wordpress to GKE
```

Comment:
@muvaf can you add this to be consistent with the crossplane.io side nav being updated for #816 

---
title: "Using GCP Services"
toc: true
weight: 420
indent: true
---

similar to https://raw.githubusercontent.com/crossplaneio/crossplane/7ecc2df467b5c0683fe257a7cc1d316b76a9445a/docs/services/azure-services-guide.md